nnclr_model.py

```diff
@@ -0,0 +1,153 @@
+""" NNCLR Model """
```

Comment:
I do not see any code about the nearest neighbour in this file, thus I do not really understand why it is called NNCLR.